streams/src/test/java/org/apache/kafka/streams/processor/internals/testutil/LogCaptureAppender.java

```diff
@@ -33,6 +34,10 @@ public static LogCaptureAppender createAndRegister() {
         return logCaptureAppender;
     }
 
+    public static void setClassLoggerToDebug(final Class<?> clazz) {
```

Comment:
:) In this case, it was forced upon me, since it's forbidden to directly configure the Logger outside of this class. (and I needed to to test the lines get logged)